### PR TITLE
Update Constitution.md

### PR DESCRIPTION
**Pull request by Marshmallows7920**

### CODE REVIEW

File: Constitution.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Computer Science Students’ Association
 The name of the student club shall be Computer Science Students’ Association, also known as CSSA.
```
Comment (lysackm) [inline Constitution.md]:
```suggestion
The name of the student club shall be Computer Science Students’ Association, also known as the CSSA.
```

File: Constitution.md
Comment (lysackm) [inline Constitution.md:1]: Some general comments about the constitution that is not related to the incoming changes:

In the definitions section of Open Club it says that Open Clubs are open to **ANY** members at U of M. We do not want to be defined as this. It states in section 1c that we are defined as an open club.

Some committees have been defined, are we defining them all them or are we just doing a select few?

File: Constitution.md [hunk 2]
```diff
@@ -77,16 +77,20 @@ Membership of the club is Free and lasts for a full academic year, September to
 
 CSSA will consist of the following executives members, who shall be voting members of the group as part of the council:
 
-- President(s)
-- Vice-President Student Affairs
-- Vice-President Internal
-- Vice-President Lounge Affairs
-- Vice-President Events
-- Interclub Liaison
+- Co-Presidents
```
Comment (lysackm) [inline Constitution.md]: Why not President(s)?

File: Constitution.md [hunk 3]
```diff
@@ -77,16 +77,20 @@ Membership of the club is Free and lasts for a full academic year, September to
...
+- Director of Finances and Internal Affairs
```
Comment (lysackm) [inline Constitution.md:81]:
```suggestion
- Executive of Finances and Internal Affairs
- Executive of Lounge Affairs
- Executive of Student Affairs
- Executive of Events
- Executive of Advocacy
```

Executive is a defined term with a meaning, while Director doesn't mean anything.

File: Constitution.md [hunk 4]
```diff
@@ -77,16 +77,20 @@ Membership of the club is Free and lasts for a full academic year, September to
...
+- Director of Events
+- Director of Advocacy
 
 CSSA will consist of the following non-executives who shall be non-voting members of the group but will be involved in assisting the executives as part of the council:
 
-- TBD
+### Councillor
+
+1. Non-voting member of council.
+2. Voting member of a committee led by a Director.
```
Comment (lysackm) [inline Constitution.md]:
```suggestion
2. Voting member of a committee led by an Executive.
```

File: Constitution.md [hunk 5]
```diff
@@ -77,16 +77,20 @@ Membership of the club is Free and lasts for a full academic year, September to
...
+3. Perform any duties directed by their committee.
 
 Voting Members may not be on academic or social probation.
```
Comment (lysackm) [inline Constitution.md]: Lets remove this line

File: Constitution.md [hunk 6]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
 
 ## Article 2b: Executive Officers and Duties
 
-### President(s)
+
+### Co-Presidents
```
Comment (lysackm) [inline Constitution.md]: This seems very disingenuous. Was this approved officially during the 2022-2023 council? It kind of felt like this was allowed through a loophole. We also don't want to allow this in the future. I don't also like that we are removing presidents for co-presidents.

File: Constitution.md [hunk 7]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
-6. Serve as signing authority on any group accounts.
-7. Be a voting member of CSSA for management decisions.
-8. Call special meetings.
-9. Attend all meetings.
-10. Carry out provisions of the constitution.
-11. Perform any other duties incidental to the office of the President or directed by Council.
+3. Maintain frequent and punctual communication with the Faculty of Science, Department of Computer Science, UMSU, Members of Council, Members of the Association, and all other relevant parties.
+4. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
+5. Be responsible for the overall operations of the group.
+6. Deal with issues concerning the club and its individual Members.
```
Comment (lysackm) [inline Constitution.md:113]: If we are trying to come up with ways to stop make it so that the presidents don't have all of the power then this feels weird. It makes it feel like the president can just penalize execs with executive authority.

File: Constitution.md [hunk 8]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+7. Serve as signing authority on any group accounts.
+8. Be a voting member of CSSA for management decisions.
+9. Call special meetings.
+10. Attend all meetings.
```
Comment (lysackm) [inline Constitution.md]: We need to define meetings because I don't think we want this once we have committees.

File: Constitution.md [hunk 9]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+2. Providing a brief report on CSSA finances and expenses at each meeting.
+3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
+4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
+5. Ensure the Council is kept updated on ongoing council initiatives.
+6. Perform a year-end audit at the end of the CSSA Fiscal Year.
+7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
+8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
+9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
+10. Ensure that the CSSA lounge is a safe and inclusive space.
+11. Lead the Financial Accounability Committee with up to 3 committee members.
```
Comment (lysackm) [inline Constitution.md:137]: Why do we have an upper limit on the number of committee members. Why not leave it open and the Finance and Internal Affairs exec can decide that?

File: Constitution.md [hunk 10]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+8. Be a voting member of CSSA for management decisions.
+9. Call special meetings.
+10. Attend all meetings.
+11. Carry out provisions of the constitution.
+12. Ensure the group funds remain in budget with local and corporate sponsors along with the Director of Finances and Internal Affairs.
+13. Ensure that the CSSA lounge is a safe and inclusive space.
+14. Enforce that the transition report is done by April 30 of the council’s term.
+15. Perform any other duties incidental to the office of the President or directed by Council.
+
+### Director of Finances and Internal Affairs
```
Comment (lysackm) [inline Constitution.md:125]:
```suggestion
### Executive of Finances and Internal Affairs
```

File: Constitution.md [hunk 11]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+6. Perform a year-end audit at the end of the CSSA Fiscal Year.
+7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
+8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
+9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
+10. Ensure that the CSSA lounge is a safe and inclusive space.
+11. Lead the Financial Accounability Committee with up to 3 committee members.
+12. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+
+### Director of Lounge Affairs
```
Comment (lysackm) [inline Constitution.md:141]:
```suggestion
### Executive of Lounge Affairs
```

File: Constitution.md [hunk 12]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+
+1. Act as a signing authority for CSSA.
+2. Create a cleaning schedule for the CSSA Lounge.
+3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
+4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
+5. Ensure that the CSSA lounge is a safe and inclusive space.
+6. Lead the CSSA lounge supervisors and work with council in appointing them.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Student Affairs
```
Comment (lysackm) [inline Constitution.md:151]:
```suggestion
### Executive of Student Affairs
```

File: Constitution.md [hunk 13]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+
+### Director of Lounge Affairs
+
+1. Act as a signing authority for CSSA.
+2. Create a cleaning schedule for the CSSA Lounge.
+3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
+4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
+5. Ensure that the CSSA lounge is a safe and inclusive space.
+6. Lead the CSSA lounge supervisors and work with council in appointing them.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
```
Comment (lysackm) [inline Constitution.md:149]: Up to Ethan but do we want to add a line about the canteen.

File: Constitution.md [hunk 14]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+
+1. Act as a liaison for the students towards the Department of Computer Science, Faculty of Science and UMSU.
+2. Act as a liaison between Students and Council.
+3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
+4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
+5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
+6. Ensure that the CSSA lounge is a safe and inclusive space.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Events
```
Comment (lysackm) [inline Constitution.md:162]:
```suggestion
### Executive of Events
```

File: Constitution.md [hunk 15]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+
+1. Be responsible for planning the events for the Computer Science community.
+2. Coordinate team-building events for Council.
```
Comment (lysackm) [inline Constitution.md]: Do we really want to make this official?

File: Constitution.md [hunk 16]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+3. Coordinate a minimum of one event or academic initiative in each term, as per the discretion of the Council.
+4. Ensure there are an adequate number of social and academic events.
+5. Create posters and advertisements for all CSSA events.
```
Comment (lysackm) [inline Constitution.md]: Again this is super specific. Is this up to the events exec or is this for the events committee?

File: Constitution.md [hunk 17]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
-9. Prepare, in cooperation with other executives of the same position when applicable, an orientation package for the subsequent person to fill each position.
-10. Attend all regular meetings.
+4. Be responsible for any specific tasks delegated to them by general resolution of Council.
+5. Declare any conflicts of interest.
+6. Declare any personal or financial interest in any expenditure of the group funds.
+7. Comply with all regulations set out by the Campus Police of the Dean’s Office, specifically those concerning smoking and drinking on campus.
+8. Promote Council cohesion and a respectful atmosphere abiding by the University of Manitoba Respectful Workplace and Learning Environment Policy.
+9. Refrain from drug/alcohol use or sexual activity at any CSSA event or in the CSSA lounge. Alcohol consumption is only permitted if the venue of the event distributes alcohol.
+10. Not allow any persons who are not Members of Council to use CSSA Office space in the absence of a Member of Council.
+11. Prepare, in cooperation with other executives of the same position when applicable, an orientation package for the subsequent person to fill each position.
```
Comment (lysackm) [inline Constitution.md:197]: There is a whole section on this, we don't need this here.

File: Constitution.md [hunk 18]
```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
...
+12. Attend all regular meetings.
+13. Put fair effort into acquiring a proxy, in the event a voting member is unable to attend a Council meeting.
 
 ## Article 2c: Executive Officer Dismissal Protocol
```
Comment (lysackm) [inline Constitution.md:201]: keep in mind of what we put in the duties in article 2b. Because technically if we don't follow it then they should go into probation.

**Review by lysackm — CHANGES_REQUESTED** [review]

File: Constitution.md (hunk 16 above)
Comment (Marshmallows7920) [reply Constitution.md]: It is something they are accountable for, but up to them to decide how they want to split or delegate their responsibility to others.

File: Constitution.md
Comment (Marshmallows7920) [reply Constitution.md:1]: defining the main ones we want to have as standard, you also mentioned on leading the outreach committee I just forgot to add that, Having that be specific to student affairs would be ok, or would student affairs have their own standard one?

File: Constitution.md (hunk 3 above)
Comment (Marshmallows7920) [reply Constitution.md:81]: The purpose is the standardized one that has been used for the class presentations along with wics and .devClub

noun
noun: director; plural noun: directors

    a person who is in charge of an activity, department, or organization.
    "he has been appointed finance director"
        a member of the board of people that manages or oversees the affairs of a business.
        
We can use other words, don't have to use director as other synonyms exist. Although some previous motions have already been signed as such, and Executive itself refers to the executive body as a whole but not specific positions. 

Requires more discussion to finalize.

File: Constitution.md (hunk 5 above)
Comment (Marshmallows7920) [reply Constitution.md]: > Lets remove this line

Line 95 about probation?
